zapier/introduction/zaps-ii/zapier-unique-search-options.md

```diff
@@ -0,0 +1,45 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# Unique Search Actions
+
+---
+## Content
+
+Up until now, we have only used Slack and Google Sheets in our zaps. Considering that there are more than 1500 possible applications in Zapier, we have used less that 1% of our possibilities (we've done the math on this).
+
+Depending on the type of application you use or integrate, you will find that each have their own actions.
+
+In this insight, we will present you the process of finding out all the possible actions and integrations, which can be used with any app.
+
+To find out what unique search actions (or triggers) an app can have, you'll first have to head on over to [Zapier](https://zapier.com/apps) and select the app in question.
+
+Let's say our stack contains Dropbox and Instagram. 
+
+![gif-of-selecting-dropbox](#)
+
+When selecting an app from the list, Zapier shows you:
+* a list of recommended integrations
+* recommended templates
+* some extra information
+* all the possible triggers, actions and searches
+
+---
+## Practice
```

Comment:
a revision question here can be inferred from this piece of content (i.e. what can you expect to see when looking at an app's page on Zapier):

```
When selecting an app from the list, Zapier shows you:
* a list of recommended integrations
* recommended templates
* some extra information
* all the possible triggers, actions and searches
```